conn.go

```diff
@@ -284,11 +296,30 @@ func (cc *Conn) FlushRuleset() {
 }
 
 func (cc *Conn) dialNetlink() (*netlink.Conn, error) {
+	var (
+		conn *netlink.Conn
+		err  error = nil
```

Comment:
There is no need to explicitly initialize variables in Go. 

Variables will always be initialized to their zero value by default.

This can just be `err error`, without the `= nil`